alica_tests/autogenerated/include/Behaviour/NotToTrigger.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
```

Comment:
I am assuming these files are just moved to the new location. No file changes other than # pragma

Reply:
:+1: